tests/src/test_environnement.cpp

```diff
@@ -8,39 +9,43 @@ TEST(Environnment, test_constructor) {
 
     EXPECT_NEAR(e.left(), -100, 0.00001);
     EXPECT_NEAR(e.right(), 100, 0.00001);
-    EXPECT_NEAR(e.bottom(), -100, 0.00001);
-    EXPECT_NEAR(e.top(), 100, 0.00001);
+    EXPECT_NEAR(e.top(), -100, 0.00001);
+    EXPECT_NEAR(e.bottom(), 100, 0.00001);
 }
 
 TEST(Environnment, test_constructor_invalid_behavior) {
     // Creates an environment of size 200m x 200m but with wrong
-    // values.
-    Environment e{100,-100,-100,100};
-
-    EXPECT_GT(e.left(), e.right());
-    EXPECT_GT(e.top(), e.bottom());
+    // catch the exception and check the message
+    try {
+        Environment e{100, -100, -100, 100};
+        FAIL() << "Expected std::invalid_argument";
+    } catch (const std::invalid_argument& err) {
```

Comment:
Connected with previous comment related to using custom class exception.

Reply:
Yes, was modify by : 
```cpp
try {
        Environment e{100, -100, -100, 100};
        FAIL() << "Expected InvalidBoundariesEnvironmentException";
    } catch (const InvalidBoundariesEnvironmentException &e) {
        std::cout << e.what() << std::endl;  // Test passed, we caught the exception
    } catch (...) {
        FAIL() << "Expected InvalidBoundariesEnvironmentException";
    }
```

in the commit [fe30d51](https://github.com/damienmarchal/junia-2024/pull/12/commits/fe30d5131872285aca82ac27dcd1c7766f8a2f99).